__tests__/integration/repeatableResources.test.js

```diff
@@ -0,0 +1,51 @@
+// Copyright 2019 Stanford University see Apache2.txt for license
+
+import Config from '../../src/Config'
+import pupExpect from 'expect-puppeteer'
+
+describe('Adding new embedded Resource Templates', () => {
+
+  beforeAll(async () => {
+    jest.setTimeout(60000); // this seems to take around 10s in practice, but be generous, just in case
+    await page.goto('http://127.0.0.1:8888/templates')
+
+    // attempt to enter and submit login info
+    try {
+      await page.waitForSelector('form.login-form')
+      await page.type('form.login-form input[name=username]', Config.cognitoTestUserName);
+      await page.type('form.login-form input[name=password]', Config.cognitoTestUserPass);
+      await page.click('form.login-form button[type=submit]');
+    } catch (error) {
+      console.info(error)
+    }
+
+    // sign out button should only show up after successful login
+    await page.waitForSelector('button.signout-btn')
+
+
+  })
+
+  it('loads up a resource template from the list of loaded templates', async () => {
+    await pupExpect(page).toClick('a', { text: 'BIBFRAME Instance' })
+    await pupExpect(page).toMatch("BIBFRAME Instance")
+
+  })
+
+  it('clicks on an AddButton in Notes about the Instance and looks for a second resource template', async () => {
+    await pupExpect(page).toClick('div.panel:nth-child(5) > div:nth-child(2) > div:nth-child(1) > div:nth-child(1) > section:nth-child(2) > div:nth-child(1) > button:nth-child(2)')
```

Comment:
this sort of selector is really tough to debug if this test stops working in the future.  

Could you use omething much more human parseable (element ids or names or classes) instead of nth-child() for all these expressions?  It might even reduce the levels of nesting needed in the selectors.

This is a practice I had to adopt for reworking the profile editor tests for utility and maintainability.

Reply:
Would it be acceptable if I ticket this for now? I don't want to spend too much today trying to figure out how to simplify this CSS selector

Reply:
sure, if you put in a todo comment linking to ticket.  :-)

Reply:
Done, new ticket #573 and noted in the code.